### PR TITLE
Paperwork rework

### PR DESCRIPTION
**Pull request by Smokingsquirrel**
## About The Pull Request
Should finally work now.

Removed SEV Torch mentions and made the texts more fitting for Eris environment.
Added Ironhammer and Guild logo.

## Screenshots
https://s19.directupload.net/images/200403/tns59mk5.jpg

### CODE REVIEW

File: code/__HELPERS/text.dm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/*
+aa/*
```
Comment (Kurgis) [inline code/__HELPERS/text.dm:1]: Loose letters.

Comment (Smokingsquirrel) [reply code/__HELPERS/text.dm]: dont know how they got in there, but uploaded a cleaned file, thank you

File: maps/CEVEris/datums/reports.dm [hunk 2]
```diff
@@ -290,63 +290,64 @@
...
 	..()
-	add_field(/datum/report_field/people/list_from_manifest, "Non-Human Employee(s) Involved")
+	add_field(/datum/report_field/people/list_from_manifest, "Non-Human(s) Involved")
 	add_signatures()
 
 /datum/computer_file/report/recipient/nt/incident/xeno_no_visa/
-	form_name = "C-3213A"
-	title = "Corporate Non-Human Employee Incident Report: Without Visa"
+	form_name = "CEV-3213A"
+	title = "Non-Human Incident Report: Without Athorization"
```
Comment (ACCount12) [inline maps/CEVEris/datums/reports.dm:316]: typo

Comment (Smokingsquirrel) [reply maps/CEVEris/datums/reports.dm]: corrected

**Pull request merged**